lib/oregon_digital/controlled_vocabularies/resource.rb

```diff
@@ -32,6 +40,21 @@ def solrize
         [rdf_subject.to_s, { label: "#{rdf_label.first}$#{rdf_subject}" }]
       end
 
+      def persist!
```

Comment:
I'm concerned about the parent/repository persistence strategy funny business that happens in ActiveTriples. I think this persists to Fedora at some point, are we sure that doesn't need to happen?

Reply:
If we are calling the method in fetch then yeah i dont think this is necessary.